### DISCUSSION TITLE
Several domains with SSO authentication

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#7311] [Self Hosting]
Hello.

Is it possible on the same instance to have several domains authorized to authenticate ?
I see I can change `ALLOWED_DOMAINS` but I don't know how I can provide several ID and SECRET in the config, one for each domain.

Best

**Comment by tommoor** [comment]
I'm not sure what you mean by ID and SECRET I'm afraid, allowed domains can be managed from the Settings -> Security page
